styles/style.css

```diff
@@ -6,3 +6,189 @@ Green: #00B172
+        background-color: white;
+        width: 1200px;
+        margin: 0 auto;
+        display: flex;
+	    flex-direction: row;
+	    flex-wrap: wrap;
+	    justify-content: flex-end;
+	    align-items: center;
+	    align-content: space-around;
+    }
+
+    .nav-bar-img {
+        align-items: left;
+        height: 80px; 
+    }
+
+/* |||||||| Header |||||||| */
+
+    .header {
+        background-image: url(/images/landing.jpg);
+        background-position: 0 0;
+        background-size: cover;
+        background-repeat: no-repeat;
+        object-fit: cover;
+        height: 850px;
+        padding-top: 0px;
+        display: flex;
+	    flex-direction: column;
+	    justify-content: center;
+	    align-items: center;
+    }
+
+    .header-title {
+        font-size: 7.3em;
+        text-transform: capitalize;
+    }
+
+    .header-text {
+        font-size: 3em;
+        color: white;
+        width: 650px;
+        text-align: center;
+    }
+
+/* |||||||| Unique Selling Points |||||||| */
+
+
+    .usp {
+        display: flex;
+        flex-direction: column;
+        justify-content: center;
+        align-items: center;
+        align-content: stretch;
+      }
+
+    .usp-img {
+        height: 90px;
+    }
+
+    .us-div-title {
+        font-size: 2.5em;
+        color: #00B172;
+    }
+
+    .usp-title {
+        font-size: 3em;
+        font-weight: 300;
+        text-align: center;
+        border-bottom: 0.35rem solid #00B172;
+    }
+
+    .usp-text {
+        font-size: 2.2em;
+        font-weight: 40;
+        width: 230px;
+
+    }
+
+    .usp-containers {
+        width: 1300px;
+        padding-top: 20px;
+        padding-bottom: 20px;
+        display: flex;
+        flex-direction: row;
+        flex-wrap: nowrap;
+        justify-content: space-around;
+        align-items: center;
+        align-content: stretch;
+    }
+
+    .usp-containers-text {
+        text-align: center;
+    }
+
+/* |||||||| App funtionalities  |||||||| */
+
+    .app-functionalities {
+        color: #FFF;
+        background-color: #00B172;
+        margin: 60px;
+        padding: 100px;
+        display: flex;
+	    flex-direction: row;
+	    flex-wrap: nowrap;
+	    justify-content: space-between;
+	    align-items: ;
+	    align-content: stretch;
+    }
+
+    .app-functionalities-logo {
+        background-image: url(/images/spotify-icon-white.png);
+        background-position: center;
+    }
+
+    .app-functionalities-title {
+        width: 370px;
+        font-size: 3.24em;
+        font-weight: 300;
+        text-align: left;
+        border-bottom: 0.3rem solid #FFF;
+    }
+
+    .app-functionalities-subtitles {
+        width: 250px;
+        font-size: 2.5em;
+        font-weight: solid;
+        text-align: left;
+        padding-top: 20px;
+    }
+
+    .app-functionalities-text {
+        width: 380px;
+        font-size: 2.3em;
+        text-align: left;
+    }
+
+
+    section:nth-of-type(2n) {
```

Comment:
Could you please explain the use of using ":nth-of-type(2n)" here?

Reply:
Two reasons:
1. I was struggling with the placement of the logo in the background. I was not able to nest it elsewhere ( the section of in its own container) and make the logo placement work.
2. Had not done it yet and  knew it was possible to do so.  

bottom line, where do you recommend I place it?

Reply:
Hey Yann,

Apologies for my late reply.
What about applying an id to the section, with the style declarations in it?
Imagine, a future you or another developer will add a section _above_ the section you are targeting. Do you see the potential risk of doing that?

Please reach out if you have a question. :)